src/Simulation/CsharpGeneration/SimulationCode.fs

```diff
@@ -1023,7 +1021,7 @@ module SimulationCode =
         | Some body ->
             let bodyName = if bodyName = "Body" then bodyName else bodyName + "Body"
             let impl =
-                ``property-arrow_get`` propertyType bodyName [``public``; ``override``]
+                ``property-arrow_get`` propertyType ("__" + bodyName + "__") [``public``; ``override``]
```

Comment:
Oh I see, you can't put the underscores with the bodyName further up because the attribute constructor needs them without underscores. That's unfortunate, but oh well.

Reply:
Sorry, I didn't realize that's what you meant. Yes, I think the __s have been moved as far up as possible.